docs/pages/access-controls/getting-started.mdx

```diff
@@ -64,10 +64,10 @@ she will be able to act as a system administrator and auditor at the same time.
 
 ## Step 2/3 Map SSO users to roles
 
-We are going to setup Github connector for OSS and Okta for Enterprises version.
+We're going to setup a Github connector for the Community Edition and Okta for Enterprise versions.
```

Comment:
According to our website (which is our current source of truth for pricing and packaging), our editions are:

https://goteleport.com/pricing

* Open Source
* Pro 
* Enterprise

Our delivery modes are:

* Self-hosted (For Open Source and Enterprise editions)
* Cloud (For Pro and Enterprise editions)